### ISSUE TITLE
Codelens fails to populate reference count in a just added VB multiproject solution

### ISSUE THREAD

**Issue by mattscheffer** [dotnet/roslyn#6014]
1. Create a new VB Console Application.
2. Add a 2nd VB console application to the project.
3. Open Module1.vb from the newly added project.

![image](https://cloud.githubusercontent.com/assets/10537189/10526332/04fd535c-733f-11e5-9178-8c68d87f8324.png)

It never changes from the above. You can still hover over the dash to see that it is 'trying' to load references but it never does and the (Alt + 2) cmd doesn't work either. 

It works again if you close and reopen the entire solution.

**Comment by mattscheffer** [comment]
@balajikris

**Comment by Pilchie** [comment]
I believe this is fixed by an open PR in the internal code for CodeLens right now.

**Comment by balajikris** [comment]
No, this is a Roslyn bug. This is one where i saw VSProjectTracker pushing projects with `null` compilationoptions to workspace hosts. Happens only for VB, happens only when adding a new or an existing project to the solution.

**Comment by balajikris** [comment]
I'll add more info..

**Comment by paul1956** [comment]
This VB only issue is covered below with a repro
```
https://social.msdn.microsoft.com/Forums/vstudio/en-US/f11101d5-f5c7-4c6d-b036-a9edb326ae7f/code-lens-doesnt-work-in-visual-studio-enterprise-2015-d14rel?forum=visualstudiogeneral
```
And reported as a confirmed bug
```
https://connect.microsoft.com/VisualStudio/feedback/details/1628295/code-lens-doesnt-work-for-vsix-application-in-vs2015
```

**Comment by balajikris** [comment]
@paul1956, thanks, I'll see if they're related and dupe this if necessary.

**Comment by balajikris** [comment]
okay the VSIX project issue is different from this.

1. Note that the VSIX repro steps is File -> New Project dialog, whereas this bug is about Adding project to existing solution. (file -> new project works fine in this bug's case)

2. In recent internal drops, the VSIX bug doesnt repro :smile: Lots of bug fixes went in codelens since vs2015 RTM and it likely got fixed.

**Comment by CyrusNajmabadi** [comment]
I don't repro this.
